src/iptvsimple/data/Channel.cpp

```diff
@@ -0,0 +1,47 @@
+#include "Channel.h"
```

Comment:
copyright header missing (please check and fix all other new files!)

Reply:
Ok, I don't have any copyright headers in the vuplus files. Should I? Do you not have to change them each year?

Reply:
.cpp and .h?

Reply:
yep